src/python/WMComponent/RucioInjector/RucioInjectorPoller.py

```diff
@@ -119,13 +93,6 @@ def __init__(self, config):
         else:
             self.isT0agent = False
 
-        if not self.listTiersToInject:
-            logging.info("Component configured to inject all the data tiers")
```

Comment:
Shouldn't this message stay, somewhere outside the `if` ?

Reply:
With the complete removal of PhEDExInjector, that message is irrelevant now. RucioInjector will always inject whatever files it has in the database pending injection.